docs/spark/how-to-guides/ubuntu-instructions.md

```diff
@@ -0,0 +1,244 @@
+---
+title: Building .NET for Apache Sapark application on Ubuntu
+description: Learn how to build your .NET for Apache Spark application on Ubuntu
+ms.date: 01/02/2020
+ms.topic: conceptual
+ms.custom: mvc,how-to
+---
+
+
+# Table of Contents
+- [Open Issues](#open-issues)
+- [Pre-requisites](#pre-requisites)
+- [Building](#building)
+  - [Building Spark .NET Scala Extensions Layer](#building-spark-net-scala-extensions-layer)
+  - [Building .NET Sample Applications using .NET Core CLI](#building-net-sample-applications-using-net-core-cli)
+- [Run Samples](#run-samples)
+
+## Open Issues:
```

Comment:
```suggestion
## Open issues

```

Reply:
These are not currently active issues, will remove this section. Thanks!